graviti/dataframe/column/series.py

```diff
@@ -502,3 +542,26 @@ def __getitem__(self, key: int) -> Optional[FileBase]:
             return None
 
         return FileBase.from_pyarrow(scalar)
+
+    @overload
+    def __setitem__(self, key: slice, value: Iterable[FileBase]) -> None:
+        ...
+
+    @overload
+    def __setitem__(self, key: int, value: FileBase) -> None:
+        ...
+
+    def __setitem__(
+        self,
+        key: Union[int, slice],
+        value: Union[FileBase, Iterable[FileBase]],
+    ) -> None:
+        try:
+            if isinstance(key, int):
+                self._data.set_item(key, value.to_pyobj())  # type: ignore[union-attr]
```

Comment:
where does the union-attr come from?

Reply:
mypy also take the type of `value` as `Union[FileBase, Iterable[FileBase]]`